src/app/components/dynamic-table/dynamic-table.component.html

```diff
@@ -8,10 +8,11 @@
                 [title]="element[disCol]"
                 mat-cell
                 class="truncate-cell"
+                [ngClass]="setOperationColumnClass(element, disCol)"
                 *matCellDef="let element"
                 [attr.data-test-id]="'column-name-' + element[disCol]"
             >
-                {{ element[disCol] }}
+                {{ disCol === "op" && dataRows?.length ? operationColumnMapper(element[disCol]) : element[disCol] }}
```

Comment:
Dynamic table is a dumb presentation-level component that should not know anything about the meaning of the data it displays. It is actually used in many scenarios unrelated to displaying data in our system, i.e. it displays schemas or environment variable.

Design a better solution where we don't have to hardcode "op" column and choose styling depending on the column name. The knowledge of what "op" means does not belong to the presentation level components. Instead, consider designing a model. where you have a higher level data row, or maybe a smarter schema column model, so that it can be given prepared data and displays it without deciding what it should display.

Reply:
Fixed